receiver/kafkametricsreceiver/metadata.yaml

```
@@ -18,17 +18,125 @@ attributes:
   group:
     description: The ID (string) of a consumer group
     type: string
+  broker:
+    description: The ID (integer) of a broker
+    type: int
 
 metrics:
+
   #  brokers scraper
   kafka.brokers:
     enabled: true
+    description: "[DEPRECATED] Number of brokers in the cluster."
+    unit: "{brokers}"
+    gauge:
+      value_type: int
+    warnings:
+      if_enabled: The metric is deprecated and will be removed. Use `messaging.kafka.broker.count`
+
+  messaging.kafka.broker.count:
+    enabled: false
     description: Number of brokers in the cluster.
     unit: "{brokers}"
```

Please update all the other units to the singular form. This is a part of the recent OTel spec guidelines

```suggestion
    unit: "{broker}"
```